Gopkg.toml

```diff
@@ -92,17 +92,17 @@ required = [
   name = "github.com/giantswarm/microendpoint"
   branch = "master"
 
-[[constraint]]
+[[override]]
```

Comment:
why `override` instead of `constraint`?

Reply:
Because I need to override the following constraints in vendored repositories:
```
Solving failure: No versions of github.com/giantswarm/microendpoint met constraints:
        master: Could not introduce github.com/giantswarm/microendpoint@master, as it has a dependency on github.com/giantswarm/microerror with constraint ^0.1.0, which has no overlap with the following existing constraints:
        master from (root)
        master from github.com/giantswarm/apprclient@master
        master from github.com/giantswarm/helmclient@master
        master from github.com/giantswarm/e2e-harness@master
        master from github.com/giantswarm/errors@master
        master from github.com/giantswarm/operatorkit@master

        go-modules: Could not introduce github.com/giantswarm/microendpoint@go-modules, as it is not allowed by constraint master from project github.com/azure-operator.
```
@fiunchinho Is there a better way of doing this?

Reply:
IMO it's ok as long as we don't have other libraries sorted.

Reply:
But I believe I fixed all those other libraries to be able to build azure-operator:
- https://github.com/giantswarm/apprclient/pull/35/files#diff-836546cc53507f6b2d581088903b1785
- https://github.com/giantswarm/helmclient/pull/175/files#diff-836546cc53507f6b2d581088903b1785

etc

Maybe I missed one of them?

Reply:
> Maybe I missed some of them?

Maybe 🤷‍♂

Reply:
Discussed on slack, marking this resolved for now.